src/algorithms/EpsilonConstraint.jl

```diff
@@ -115,5 +115,5 @@ function optimize_multiobjective!(
         rhs += ε
     end
     MOI.delete(model, ci)
-    return MOI.OPTIMAL, unique(solutions)
+    return MOI.OPTIMAL, filter_nondominated(solutions)
```

Comment:
For the epsilon constraint, filtering could be done after each call to optimize! instead.

See https://github.com/vOptSolver/vOptGeneric.jl/blob/4b0bbba009330271938a1973a3da7f32556ec6d4/src/algorithms.jl#L103

Reply:
Simpler here for now. We can always revisit later.